protoc-gen-grpc-gateway/descriptor/services.go

```diff
@@ -205,6 +214,18 @@ func extractAPIOptions(meth *descriptor.MethodDescriptorProto) (*options.HttpRul
 	return opts, nil
 }
 
+func defaultAPIOptions(svc *Service, md *descriptor.MethodDescriptorProto) (*options.HttpRule, error) {
+	// FQSN prefixes the service's full name with a '.', e.g.: '.example.ExampleService'
+	fqsn := strings.TrimPrefix(svc.FQSN(), ".")
+	rule := &options.HttpRule{
+		Pattern: &options.HttpRule_Post{
+			Post: fmt.Sprintf("/%s/%s", fqsn, md.GetName()),
+		},
+		Body: "*",
+	}
```

Comment:
Could we add a reference to the gRPC HTTP/2 spec here (https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#requests), so that we know what we're basing this default pathing on?